vcr/issuer/leia_store.go

```diff
@@ -89,7 +89,7 @@ func NewLeiaIssuerStore(dbPath string, backupStore stoabs.KVStore) (Store, error
 }
 
 func (s leiaIssuerStore) StoreCredential(vc vc.VerifiableCredential) error {
-	vcAsBytes, _ := json.Marshal(vc)
+	vcAsBytes, _ := json.Marshal(types.CompactingVerifiableCredential(vc))
```

Comment:
add comment on why it is needed. (To prevent copy pasta when migrating to a new DB where I assume it is no longer needed?)

Reply:
Added comment. Your assumption is right.